packages/flutter/lib/src/painting/image_stream.dart

```diff
@@ -510,7 +510,8 @@ class MultiFrameImageStreamCompleter extends ImageStreamCompleter {
        _informationCollector = informationCollector,
        _scale = scale,
        _framesEmitted = 0,
-       _timer = null {
+       _timer = null,
+       _frameCallbackScheduled = false {
```

Comment:
nit: It reads a little cleaner to me if you would just initialize the member with false in line 539 instead of doing it here.

Reply:
done